src/Http/Http.Extensions/src/HttpValidationProblemDetails.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Microsoft.AspNetCore.Http.Extensions
+{
+    /// <summary>
+    /// A <see cref="ProblemDetails"/> for validation errors.
+    /// </summary>
+    [JsonConverter(typeof(HttpValidationProblemDetailsJsonConverter))]
+    public class HttpValidationProblemDetails : ProblemDetails
```

Comment:
sealed?

Reply:
```diff
namespace Microsoft.AspNetCore.Mvc
{
-   public class ValidationProblemDetails : ProblemDetails
+   public class ValidationProblemDetails : HttpValidationProblemDetails
    {
    }
}
```

Reply:
I'm open to not doing this - it just felt like a good way to help move the feature over to Http.Extensions.

Reply:
I was originally thinking we would have ProblemDetails derive from something in the core ProblemDetailsBase.